src/mpid/ch3/src/mpid_init.c

```diff
@@ -244,27 +276,13 @@ int MPID_Init(int requested, int *provided)
     MPIR_ERR_CHECK(mpi_errno);
 
   fn_exit:
-    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPID_INIT);
+    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_MPID_INIT_WORLD);
     return mpi_errno;
-
-    /* --BEGIN ERROR HANDLING-- */
   fn_fail:
     goto fn_exit;
     /* --END ERROR HANDLING-- */
 }
```

Comment:
`/* --BEGIN ERROR HANDLING-- */` is removed but `/* --END ERROR HANDLING-- */` is still here.

Reply:
I'll clean it up.

Reply:
fixed. (I undeleted the "BEGIN..." so we have one less extra change).